api/src/main/java/com/github/eyefloaters/console/api/Annotations.java

```diff
@@ -0,0 +1,26 @@
+package com.github.eyefloaters.console.api;
+
+public enum Annotations {
+
+    /**
+     * Annotation to identify a listener in Strimzi Kafka resources to be used for
+     * connections directly from the Console API.
+     */
+    CONSOLE_LISTENER("console-listener");
```

Comment:
I guess my only other thought about this is: When do we start writing some user-centric documentation about how to deploy what we're writing. It's the sort of thing where what you need to document quickly mounts up. Right now putting something in the `README.md` would be sufficient imho.

Reply:
> I guess my only other thought about this is: When do we start writing some user-centric documentation about how to deploy what we're writing. It's the sort of thing where what you need to document quickly mounts up. Right now putting something in the `README.md` would be sufficient imho.

Added #63 to track this.